### PR TITLE
[SW-369] Node connection statuses as constants

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Review by pstachel — APPROVED** [review]

**Review by konradodo — APPROVED** [review]

File: src/popup/components/Modals/TransferReceive.vue [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
       <AccountRow />
 
+      <div style="height: 400px;" />
```
Comment (CedrikNikita) [inline src/popup/components/Modals/TransferReceive.vue:18]: Why did you start to use `style` attribute?

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
ERROR in `...superhero-wallet/src/popup/App.vue(95,27)`:
95:27 Cannot find name 'browser'.

**Comment by CedrikNikita** [comment]
Also your refactoring is completely changing the design of networks page, where the goal was to show the connection status right underneath the network name.

File: src/popup/components/Modals/TransferReceive.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/Modals/TransferReceive.vue]: This was an element that allowed me to test scrolling on the modals. I removed this.

**Comment by CedrikNikita** [comment]
The popup doesn't have connection status and both buttons `deny` `accept` are missaligned.
image

**Review by konradodo — APPROVED** [review]

**Review by CedrikNikita — APPROVED** [review]
Code changes look reasonable.

**Pull request merged**